R/apa_print_emm_lsm.R

```diff
@@ -52,7 +53,7 @@ apa_print.summary_emm <- function(
   x
   , contrast_names = NULL
   # , stat_name = NULL
-  , est_name = NULL
+  , est_name = "\\widehat{\\theta}"
```

Comment:
I always preferred `\hat` over `\widehat` for `\theta`, think, wide hat is too wide